firmware/uf2.ksy

```diff
@@ -0,0 +1,169 @@
+meta:
+  id: uf2
```

Comment:
maybe `usb_flashing_format`. `uf2` is too short, there may be a collision.

Reply:
UF2 is the most well known name for this format. I did a few queries with several search engines and all the first hits are for this particular format, so I am not too worried about collisions.